src/content/docs/de/core-concepts/astro-syntax.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: Astro Syntax
+description: Eine Einführung in die Syntax der .astro-Komponenten 
+i18nReady: true
+---
+
+**Wenn du HTML kennst, weißt du bereits genug, um deine erste Astro-Komponente zu schreiben.**
+
+Die Syntax von Astro-Komponenten ist eine Erweiterung von HTML. Die Syntax wurde [so entworfen, dass sie für jeden, der Erfahrung im Schreiben von HTML oder JSX hat, vertraut wirkt](#unterschiede-zwischen-astro-und-jsx), und sie ermöglicht das Einbinden von Komponenten und JavaScript-Ausdrücken.
+
+## JSX-ähnliche Ausdrücke
+
+Du kannst lokale JavaScript-Variablen innerhalb des Skripts der Frontmatter-Komponente zwischen den beiden Code-Grenzen (`---`) einer Astro-Komponente definieren. Du kannst dann diese Variablen mithilfe von JSX-ähnlichen Ausdrücken in das HTML-Template der Komponente einfügen!
```

Comment:
Wollen wir Code-Grenzen in übersetzter Form oder dort bei englisch bleiben?

Reply:
Ich finde Code-Grenzen ist in Ordnung.